awesomemotive/wp-mail-logging
# UI improvements

## Description

This PR adds UI improvements to the email logs table.

## Motivation

Fixes #155.

## Testing procedure

1. Navigate to Dashboard -> WP Mail Logging -> Email Logs.
2. Hover your mouse to the "view", "resend", and "delete" icons. You should see tooltips on them.

<img width="133" alt="Screen Shot 2023-06-16 at 21 06 23" src="https://github.com/awesomemotive/wp-mail-logging/assets/5747475/dadbdd94-1fba-499f-a38f-e33facd3c98a">

<img width="148" alt="Screen Shot 2023-06-16 at 21 06 31" src="https://github.com/awesomemotive/wp-mail-logging/assets/5747475/4303b896-a7f8-4e27-9ba9-92b709047a51">

<img width="147" alt="Screen Shot 2023-06-16 at 21 06 39" src="https://github.com/awesomemotive/wp-mail-logging/assets/5747475/819ef1f2-e65f-47df-9653-20f104183db0">

3. Click the "delete" icon. You should see a confirm dialog box to confirm the delete action.

<img width="425" alt="Screen Shot 2023-06-16 at 21 07 24" src="https://github.com/awesomemotive/wp-mail-logging/assets/5747475/c4c67f0d-e1a5-4298-b1f8-f41a739fa0af">
